stylix/palette.html.mustache

```diff
@@ -63,6 +63,6 @@
 
         <h2>Documentation</h2>
         <p>Each color should be used as described in <a href="https://github.com/chriskempson/base16/blob/main/styling.md#styling-guidelines"> this table</a>.</p>
-        <p>See the <a href="https://stylix.danth.me/">Stylix documentation</a> for how to apply these colors on NixOS.</p>
+        <p>See the <a href="https://nix-community.github.io/stylix/">Stylix documentation</a> for how to apply these colors on NixOS.</p>
```

Comment:
```suggestion
        <p>See the <a href="https://nix-community.github.io/stylix">Stylix documentation</a> for how to apply these colors on NixOS.</p>
```
Ah, I would drop the final slash, for consistency with the other instances of the base URL.
I’d also suggest doing the same on the main page (https://github.com/nix-community/stylix), where it is shown on the right side, under the **About** widget. Or the other way around, if that’s the preferred style, ofc.